Shared/PathUtils.cs

```diff
@@ -131,5 +131,24 @@ public static string GetTempFileName(string extension)
             //use instead of Path.GetTempFileName()
             return Path.Combine(Path.GetTempPath(), Path.GetRandomFileName() + extension);
         }
+
+        /// <summary>
+        /// Checks if the relative path has any OpenTapIgnore in parent chain
+        /// </summary>
+        /// <param name="location">Relative path of file</param>
+        /// <returns>The precense of OpenTapIgnore in parent chain</returns>
+        internal static bool DecendsFromOpenTapIgnore(string location)
+        {
+            string dir = Path.GetDirectoryName(location);
+            if (File.Exists(Path.Combine(dir, ".OpenTapIgnore")))
+                return true;
+            if (string.IsNullOrWhiteSpace(dir))
+                return false;
+            DirectoryInfo d = new DirectoryInfo(dir);
+            if (d.Parent == null)
```

Comment:
I dont think we need to add this check, isn't the final condition catched by string.IsNUllOrWhiteSpace(dir)?

maybe just add: if (string.IsNullOrWhiteSpace(dir) || dir == "/" || dir == location) return fals